### DISCUSSION TITLE
Zlibrary.msi安装后兼容性问题

### DISCUSSION THREAD

**Discussion by Nexro920** [tw93/Pake#313] [Q&A]
版本：V1.0.4-beta
文件：Zlibrary_x64.msi
环境：win10
过程：正常下载msi并安装后，在打开程序时，提示“兼容性问题”
详细提示：一个不兼容的软件尝试与Microsoft Edge 一起加载。
错误代码：STATUS_INVALID_IMAGE_HASH

期间也曾按照网上解决方案在注册表里增加了Edge项并添加RendererCodeIntegrityEnabled=0，但仍无法解决，请问这种情况该如何处理。

**Comment by Tlntin** [comment]
win10版本号？
类型？：例如教育/专业/家庭版/专业工作站/企业版/政府专用版？
